src/iperf_server_api.c

```diff
@@ -243,11 +244,11 @@ iperf_handle_message_server(struct iperf_test *test)
                 close(sp->socket);
             }
             test->reporter_callback(test);
-	    if (iperf_set_send_state(test, EXCHANGE_RESULTS) != 0)
+            if (iperf_set_send_state(test, EXCHANGE_RESULTS) != 0)
```

Comment:
The indentation changes look good. There are also 3 places where the change adds some braces in `if` statements. Those brace changes are also fine, but the commit message makes it look like the only changes are just whitespace. Could you amend the commit message to reflect that you're adding braces to a few `if` statements as well?

Reply:
And that wasn't the right place to make this comment. Lines 517, 522, and 529.